bindings/nodejs/lib/client/client.ts

```diff
@@ -604,6 +761,19 @@ export class Client {
         return plainToInstance(UTXOInput, parsed.payload);
     }
 
+    // Other routes.
+
+    /**
+     * Get the network related information such as network_id.
```

Comment:
This just returns protocol parameters now, let's remove it instead, since we have `GetProtocolParameters`?

Reply:
makes sense

Reply:
Removed